css-values/Overview.bs

```diff
@@ -1304,16 +1618,26 @@ Duration Units: the <<time>> type and ''s'', ''ms'' units</h3>
 -->
 
 <h3 id="frequency">
-Frequency Units: the <<frequency>> type and ''Hz'', ''kHz'' units</h3>
+Frequency Units: the <<frequency>> type and 
+''Hz'', ''kHz'', 
+''bpm''
+units</h3>
 
 	Frequency values are <a>dimensions</a> denoted by <dfn id="frequency-value">&lt;frequency></dfn>.
 	The frequency unit identifiers are:
 
 	<dl export dfn-type=value dfn-for="<frequency>">
-	  <dt><dfn id="Hz">Hz</dfn>
+
+	  <dt><dfn id="Hz">Hz unit</dfn>
 	  <dd>Hertz. It represents the number of occurrences per second.
-	  <dt><dfn id="kHz">kHz</dfn>
-	  <dd>KiloHertz. A kiloHertz is 1000 Hertz.
```

Comment:
proper spelling in English is lowercase, SI unit symbol is has initial uppercase letter.